This reverts enabling powerpc related code on little endian ppc64 as it
seems to contain some big endian related assumptions, resulting in SIGILLs
on little endian.

--- m4/as-host-defines.m4
+++ m4/as-host-defines.m4
@@ -19,7 +19,7 @@ AC_DEFUN([AS_HOST_DEFINES],
       HAVE_AMD64=yes
       AC_DEFINE(HAVE_AMD64, 1, [Defined if host is amd64])
       ;;
-    xpowerpc|xppc|xpowerpc64*|xppc64*)
+    xpowerpc|xppc|xpowerpc64|xppc64)
       HAVE_POWERPC=yes
       AC_DEFINE(HAVE_POWERPC, 1, [Defined if host is powerpc])
       ;;
